Tasks/CopyFilesOverSSHV0/copyfilesoverssh.ts

```diff
@@ -1,132 +1,127 @@
-import os = require('os');
-import path = require('path');
-import Q = require('q');
-
-import tl = require('vsts-task-lib/task');
-import sshCommon = require('ssh-common/ssh-common');
-import {SshHelper} from 'ssh-common/ssh-common';
-
-//This method will find the list of matching files for the specified contents
-//This logic is the same as the one used by CopyFiles task except for allowing dot folders to be copied
-//This will be useful to put in the task-lib
-function getFilesToCopy(sourceFolder, contents: string []) : string [] {
+import * as os from 'os';
+import * as path from 'path';
+
+import * as tl from 'vsts-task-lib/task';
+import { SshHelper } from 'ssh-common/ssh-common';
+
+// This method will find the list of matching files for the specified contents
+// This logic is the same as the one used by CopyFiles task except for allowing dot folders to be copied
+// This will be useful to put in the task-lib
+function getFilesToCopy(sourceFolder, contents: string[]): string[] {
```

Comment:
It looks like the two work a little differently right now, so this will be a behavior change: https://buildcanary.visualstudio.com/CanaryBuilds/CanaryBuilds%20Team/_build/results?buildId=71763&view=logs

Reply:
ok lets leave that change for a major update then!